sos/report/plugins/systemd.py

```diff
@@ -39,6 +39,7 @@ def setup(self):
             # status --all mostly seems to cover the others.
             "systemctl list-units",
             "systemctl list-units --failed",
+            "systemctl list-units --all",
```

Comment:
I am OK to add the `--all`. Just thinking about possible duplication data collection: is there some reason to collect similar info three times (`list-units`, `--failed` and `--all`)? Are there unique pieces of information in each of the command outputs?

Reply:
Disregard, somehow answered already.